### PR TITLE
docs : Update README.md

### LINKED ISSUE #557: Update readme
**Summary:**

i want to add readme.md. As it has very less information about the project and setup

### PR DESCRIPTION
**Pull request by RajaVamsi11**
Fixes #557 

Updated Readme file so that new contributors can find easy to start contributing. You can check [here](https://github.com/RajaVamsi11/mifos-mobile/tree/UpdateReadMe) how updated README looks like. I took reference from the read me file of the mobile wallet project.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by RajaVamsi11** [comment]
@miPlodder @therajanmaurya Can you please review this PR.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,64 @@
+<img height='175' src="https://user-images.githubusercontent.com/37406965/51083189-d5dc3a80-173b-11e9-8ca0-28015e0893ac.png" align="left" hspace="1" vspace="1">
+
+# Self Service Android Application for MifosX
```
Comment (therajanmaurya) [inline README.md]: We have changed the name Self Service -> Mifos Mobile, So It should be  Mifos-Mobile Android Application for MifosX

File: README.md [hunk 2]
```diff
@@ -1,16 +1,64 @@
...
 
-An Android Application built on top of the MifosX Self-Service platform for end-user customers to view/transact on the accounts and loans they hold. Data visible to customers will be a sub-set of what staff can see. This is a native Android Application written in Java with POJO classes in Kotlin.
+<p>
+  <img src="https://user-images.githubusercontent.com/24931732/48102172-b0ccc800-e1de-11e8-9fb9-17c41234636e.png" />
+</p>
+
+## How to Contribute
+
+This is an OpenSource project and we would be happy to see new contributors. The issues should be raised via the GitHub issue tracker.
+For Issue tracker guidelines please click <a href="https://github.com/openMF/mifos-mobile/blob/development/.github/CONTRIBUTING.md">here</a>. All fixes should be proposed via pull requests.
```
Comment (therajanmaurya) [inline README.md]: Fix the Issue template link: https://github.com/openMF/mifos-mobile/blob/development/.github/ISSUE_TEMPLATE.md

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by RajaVamsi11** [comment]
@therajanmaurya Updated according to the changes suggested.

**Review by therajanmaurya — APPROVED** [review]

**Pull request merged**